charts/fybrik/kind-control.values.yaml

```diff
@@ -26,14 +26,15 @@ cluster:
 
 # Configuration when deploying to a coordinator cluster.
 coordinator:
+  catalogConnectorURL: "http://katalog-connector:80"
```

Comment:
why does the default `catalogConnectorURL` value need to be changed?

Reply:
This is not a mock setup. `make deploy` is used. Value of catalogConnectorURL should not change, it is defined in charts/fybrik/values.yaml. Since katalog-connector is not open-api based yet, adding "http://" won't work.

Reply:
Refer to https://github.com/fybrik/fybrik/blob/ce6d8e7cdf27b20d7b683126aeea3af583c2dc97/charts/fybrik/kind-control.values.yaml#L54-L56 , in this setup mock is used in the same file and the mock does not have grpc endpoint, we should make them consistent.

Reply:
If not add the "http://" prefix the manager will keep restarting.

Reply:
Ok, you are right, I missed the service image generation.